odl/operator/pspace_ops.py

```diff
@@ -365,14 +366,16 @@ def __getitem__(self, index):
 
         Parameters
         ----------
-        index : tuple of int
+        index : int or tuple of int
             A pair of integers given as (row, col).
 
         Returns
         -------
-        suboperator : `Operator` or ``0``
-            If there is an operator at (row, col), the operator is returned,
-            otherwise ``0``.
+        suboperator : `ReductionOperator`, `Operator` or ``0``
```

Comment:
Personally I'd say yes, mostly because it typesets and thus it is more clear that we mean the python value 0 instead of just the "mathematical concept 0", if you see what I mean.